src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/service/Functions.kt

```diff
@@ -3,3 +3,16 @@ package uk.gov.justice.digital.hmpps.hmppsassessforearlyreleaseapi.service
 import org.springframework.core.ParameterizedTypeReference
 
 inline fun <reified T> typeReference() = object : ParameterizedTypeReference<T>() {}
+
```

Comment:
All these were just copied from CVL. I can't see anyway of sharing stuff like this between projects at the moment. It looks like kotlin-lib is just for spring stuff etc.

Reply:
not super worried about duplication for things like this..